jest.config.js

```diff
@@ -14,4 +14,5 @@ module.exports = {
 		]
 	],
 	verbose: true,
+	silent: true,
```

Comment:
Hmm. This could be a big gotcha in the making e.g. I'm trying to debug a failing test by adding in a `console.log` but no output is showing and I have no idea why.

What's currently console logging?

Reply:
Given that `--silent` and `--verbose` are both options on the CLI. I'd like to suggest that we are not prescriptive with either in our configuration.

Reply:
No worries.